src/logger.cpp

```diff
@@ -11,13 +11,17 @@ Logger::Logger(const QString& module, const QString& className)
 Logger::Logger(const QStringList& modules, const QString& className)
     : m_modules(modules), m_className(className) {}
 
-Logger::Log Logger::log() { return Log(this); }
```

Comment:
I'd also like to see a Logger::info() in between `warning()` and `debug()` for messages that we think should be printed to console even though they might not be an error (eg: login banners, major state changes and etc). I find it kind of unsettling that the console is completely silent on production builds. I prefer a little sign of life in my applications.